src/noise.cpp

```diff
@@ -394,14 +394,14 @@ void Noise::allocBuffers()
 	this->noise_buf = NULL;
 	resizeNoiseBuf(sz > 1);
 
-	delete[] gradient_buf;
+	delete[] value_buf;
 	delete[] persist_buf;
 	delete[] result;
 
 	try {
 		size_t bufsize = sx * sy * sz;
 		this->persist_buf  = NULL;
-		this->gradient_buf = new float[bufsize];
+		this->value_buf = new float[bufsize];
```

Comment:
nitpick: the lines surrounding this aren't aligned anymore after this, the alignment should either be kept or gotten rid of entirely

(we should also probably update our code style guidelines. i don't like "aligning" code because inevitably renames break it.)